code/src/index.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import './index.css'
-import { App } from './App'
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import {App} from "./App";
```

Comment:
The order should be (according to Maks)

import React from "react";
import ReactDOM from "react-dom";

import {App} from "./App";

import "./index.css";

Reply:
👍